projects/cm_mcu/I2CCommunication.c

```diff
@@ -144,11 +144,11 @@ int apollo_i2c_ctl_reg_r(uint8_t address, uint8_t reg_address, uint8_t nbytes, u
   return 0;
 }
 
+#define MAX_BYTES 4
 int apollo_i2c_ctl_reg_w(uint8_t address, uint8_t reg_address, uint8_t nbytes, uint8_t packed_data)
```

Comment:
I thought `packed_data` should also be changed to a 32 bit integer since you are shifting it down by up to 24 bits?

Reply:
might not affect the current use case but sooner or later it will cause an issue?

Reply:
changed both packed_data and value to int in commit 76737d2